addons/delivery/migrations/9.0.1.0/post-migration.py

```diff
@@ -26,6 +26,44 @@ def rename_property(cr, model, old_name, new_name):
         (new_name, field_ids))
 
 
+def reassign_carrier_id(cr):
+    print("""Delivery carrier ids have changed""")
+    update_sale = False
+    cr.execute("""
+        SELECT table_name
+        FROM information_schema.tables
+        WHERE table_name='sale_order'
+    """)
+    if cr.fetchone():
+        update_sale = True
+
+    update_stock_picking = False
+    cr.execute("""
+        SELECT table_name
+        FROM information_schema.tables
+        WHERE table_name='stock_picking'
+    """)
+    update_stock_picking = False
```

Comment:
delete this line because it's repeated

Reply:
It's not needed directly using `openupgrade.column_exists`